app/services/api/v0_1/create_hesa_trainee_detail_service.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Api
+  module V01
+    class CreateHesaTraineeDetailService
+      include ServicePattern
+
+      def initialize(trainee:)
+        @trainee = trainee
+      end
+
+      def call
+        return true unless trainee.hesa_trainee_detail.nil?
```

Comment:
It might be confusing to return `true` since the service won't create a new record. Should we return `false` instead?